packages/web/src/components/spa/HackAUBG/VideoSection/video_section.css

```diff
@@ -23,20 +20,38 @@
     justify-content: center;
     align-items: center;
     display: flex;
-    padding: 3% 0;
     padding: 3% 0%;
-    text-align: justify;
+    text-align: center;
 }
 
 @media only screen and (max-width: 900px) {
+    .video-container {
+        height: 65vh;
+    }
+
     .video-header {
-        font-size: 0.5rem;
-        text-align: justify;
+        font-size: 1.5em;
+        text-align: center;
     }
     .video {
-        width: 80%;
+        padding-left: 10%;
+        padding-right: 10%;
         justify-content: center;
         align-items: center;
         display: flex;
+        height: 49vh;
+        width: 100%;
+    }
+}
+
+@media only screen and (max-width: 1300px) {
+    .video-header {
+        font-size: 2.5em;
+    }
+}
+
```

Comment:
Why are there two media queries with the same max-widths. You can combine them in one. And also for some reason the text is too big and not responsive on mobile devices.